Project.toml

```
@@ -1,20 +1,22 @@
 name = "HTML_Entities"
-desc = "Entities from HTML data tables"
-authors = ["ScottPJones <scottjones@alum.mit.edu>"]
+uuid = "7693890a-d069-55fe-a829-b4a6d304f0ee"
 keywords = ["Entities", "HTML"]
 license = "MIT"
-uuid = "7693890a-d069-55fe-a829-b4a6d304f0ee"
+desc = "Entities from HTML data tables"
+authors = ["ScottPJones <scottjones@alum.mit.edu>"]
 version = "1.0.1"
 
 [deps]
+RelocatableFolders = "05181044-ff0b-4ac5-8273-598c1e38db00"
 StrTables = "9700d1a9-a7c8-5760-9816-a99fda30bb8f"
 
+[compat]
+RelocatableFolders = "0.3"
```

If using `RelocatableFolders` for this one, I'd suggest going with `1` compat. Looks like this PR was made at pretty much the exact same time as version 1.0.0 was released. No difference in exposed API, it's just considered feature-complete and stable.

```suggestion
RelocatableFolders = "1"
```